cmd/k8s-bigip-ctlr/main.go

```diff
@@ -183,6 +184,8 @@ func _init() {
 		"Optional, when set to false, disables as3 template validation on the controller.")
 	sslInsecure = bigIPFlags.Bool("insecure", false,
 		"Optional, when set to true, enable insecure SSL communication to BIGIP.")
+	logResponseBody = bigIPFlags.Bool("log-response-body", false,
+		"Optional, when set to true, add the body of AS3 API calls in Controller logs.")
```

Comment:
is that, AS3 API call or response ?

Reply:
This is only for AS3 API call.

Reply:
Optional, when set to true, add the body of AS3 API response in Controller logs.